store/store.go

```diff
@@ -190,21 +190,35 @@ type ChannelMemberHistoryStore interface {
 	PermanentDeleteBatch(endTime int64, limit int64) (int64, *model.AppError)
 }
 
+type GetPostsSinceOptions struct {
```

Comment:
Can we move that to models? is where they are and if not probably you are breaking things like the mocks and the timer layer.

Reply:
Anyway, I'm not 100% onboard with this ValueObject approach here, I get the point of this, but is changing the approach used in the rest of the store API, if we going to change it, probably we should have a conversation before, and decide if we want to follow that path or not.